### ISSUE TITLE
(minor)

### ISSUE THREAD

**Issue by TBody** [scipy/scipy#11505]
Using the new version of scipy.integrate, setting an array for `rtol` in `solve_ivp` results in an error.

```
solve_ivp(..., rtol=[1,2,3,4,5], ...)
```
will return
```
scipy/integrate/_ivp/common.py", line 46, in validate_tol
    if rtol < 100 * EPS:
ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()
```

Modifying the line to be `if np.any(rtol < 100 * EPS` seems to resolve the issue

**Comment by tylerjereddy** [comment]
You're saying this used to work in a previous version of SciPy but no longer does now?

**Comment by TBody** [comment]
Not exactly: I think that `solve_ivp` uses different methods to `scipy.integrate.ode` (marked as old API in new version of scipy). The method using `scipy.integrate.ode` (using `ode(f, jac).set_integrator('dop853', rtol=[1,2,3,4,5],...`) worked without issue.

**Comment by drhagen** [comment]
I confirmed that [dop853](https://github.com/scipy/scipy/blob/master/scipy/integrate/dop/dop853.f#L46-L4) allowed for a vector rtol. However, [others](https://github.com/scipy/scipy/blob/master/scipy/integrate/odepack/lsoda.f#L66) did not. I have thought about this feature in the past, not because I needed it, but because it seemed like it is straightforward to implement. I did not know that any ODE solver implementation had done this. The thing is, I don't know how valid this is, especially for the algorithms that did not have it before. Does having a different `rtol` on each state break any of the assumptions underlying the math of the ODE method?

**Comment by srossell** [comment]
Hi,

I have run into the same issue with solve_ivp. The same model works accepting vectors for atol and rtol in scipy.integrate.odeint. I'm using the BDF solver.

I'm surprised to see that this issue is tagged as minor. The possibility of setting tolerances for each state variable is truly useful, even essential for difficult problems.
